Makefile

```diff
@@ -138,9 +138,9 @@ ifeq ($(APT_BASED),0)
 # yq is necessary for some scripts that process controller-info yaml files.
 	@$(call check_dep,yq,Missing yq - install with 'sudo snap install yq')
 # Microk8s is required if you want to start a Juju controller on Microk8s.
-	@$(call check_dep,microk8s,Missing microk8s - install with 'sudo snap install microk8s')
+	@$(call check_dep,microk8s,Missing microk8s - install with 'sudo snap install microk8s --classic')
```

Comment:
Since JIMM v3 works with with Juju 3.X, it should be an strict snap installation. For example:

```sh
sudo snap install microk8s --channel=1.30-strict/stable
```